presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveFileFormats.java

```diff
@@ -376,6 +377,24 @@ public void testAvro(int rowCount)
                 .isReadableByRecordCursor(createGenericHiveRecordCursorProvider(HDFS_ENVIRONMENT));
     }
 
+    @Test(dataProvider = "rowCount")
+    public void testSymlinkAvroFile(int rowCount)
```

Comment:
This doesn't test symlinks. 

`testAvroFileInSymlinkTable` ?

Reply:
Would be good to also have an end-to-end test. `TestHiveIntegrationSmokeTest` should be a good place